packages/data-explorer/src/components/SearchComponent.vue

```diff
@@ -64,12 +65,12 @@ export default {
     },
     searchText: {
       // Add handler to support html5 clear search action
-      handler: function (val, oldVal) {
+      handler: function (val) {
         if (val === '') {
           this.handleSearchAction()
         }
       },
-      immediate: true
+      immediate: false
```

Comment:
why is searchText no longer evaluated immediate ?

Reply:
Reverted changes